src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -100,9 +102,7 @@ class FagsakService(
     }
 
     fun hentMinimalFagsakForPerson(personident: String): MinimalFagsakResponsDto {
-        val aktør = personidentService.hentOgLagreAktør(personident, true)
-        val fagsak = fagsakRepository.finnFagsakForAktør(aktør) ?: throw Feil("Fant ikke fagsak på person")
-        return lagMinimalFagsakResponsDto(fagsak = fagsak)
+        return lagMinimalFagsakResponsDto(hentFagsakForPerson(personident))
     }
```

Comment:
```suggestion
       fun hentMinimalFagsakForPerson(personident: String): MinimalFagsakResponsDto = 
       lagMinimalFagsakResponsDto(hentFagsakForPerson(personident))
```

Siden vi inliner variablene nå, så kan vi bytte til single expression

Reply:
Tenker at vi kan kjøre på med det samme for de andre nye metodene i denne klassen